WBooks/Book Comments/BookCommentsViewController.swift

```diff
@@ -33,10 +33,22 @@ class BookCommentsViewController: UIViewController {
     }
     
     private func configureCommentsTableView() {
-        _viewModel.comments.producer.startWithValues { [weak self] _ in
-            self?._view.comments.reloadData()
-            self?._view.stopActivityIndicator()
+        _viewModel.state.producer.startWithValues { state in
+            switch state {
+            case .loading:
+                break
+            case .error:
+                self._view.stopActivityIndicator()
+                self._view.displayCommentsIndicator(state: self._viewModel.state.value)
+            case .empty:
+                self._view.stopActivityIndicator()
+                self._view.displayCommentsIndicator(state: self._viewModel.state.value)
```

Comment:
for `.error` and `.empty` you are doing the same calls, so you could set this as: `case .error, .empty`